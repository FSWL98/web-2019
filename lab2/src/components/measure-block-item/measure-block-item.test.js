import React from "react";
import MeasureBlockItem from "./measure-block-item";
import renderer from 'react-test-renderer';


it("Basic MeasureBlockItem", () => {
    const component = renderer.create(<MeasureBlockItem />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

it("Blue MeasureBlockItem", () => {
    const component = renderer.create(<MeasureBlockItem color={"blue"}/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
