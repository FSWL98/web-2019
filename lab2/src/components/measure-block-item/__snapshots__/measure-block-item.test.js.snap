// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic MeasureBlockItem 1`] = `
<div
  className="measure-block-item measure-block-item--gray d-flex mt-md-1 justify-content-between row"
>
  <div
    className="col"
  />
  <div
    className="d-flex justify-content-end font-weight-light col"
  />
</div>
`;

exports[`Blue MeasureBlockItem 1`] = `
<div
  className="measure-block-item measure-block-item--blue d-flex mt-md-1 justify-content-between row"
>
  <div
    className="col"
  />
  <div
    className="d-flex justify-content-end font-weight-light col"
  />
</div>
`;
